CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 3.8.1)
+project(QUICK NONE)
+
+message("before init")
+include(cmake/QuickBuildSystemInit.cmake NO_POLICY_SCOPE)
+message("before enable_language")
+enable_language(Fortran C CXX) #now ENABLED_LANGUAGES is a property but not a variable
```

Comment:
CUDA should be included in this list, see my comment on FindCUDA.

Reply:
`CUDA` has been added in this list.